test/viewmodels/SecuritiesRequestedStatementsViewModelSpec.scala

```diff
@@ -262,7 +261,4 @@ trait SpecBaseWithSetup extends SpecBase {
     SecuritiesRequestedStatementsViewModel(securityStatements)
   }
 
-  when(messages("cf.account.details.previous-eori", eoriNumber))
```

Comment:
Why this has been deleted?

Reply:
mocking was redundant, without mocking the test case works fine without troubles